src/frame.test.tsx

```diff
@@ -125,4 +133,206 @@ describe("adding elements to frames", () => {
       // );
     });
   });
+
+  /**
+   * TODO:
+   * dragging element into the frame
```

Comment:
Updated the algo, so resizing the frame should efficiently reorder elements as expected (please check the test cases below).

Dragging elements into the frame still does not work, as it is influenced by some later step (possibly by `updateFrameMembershipOfSelectedElements` or `getElementsInResizingFrame`).